cves/CVE-2011-3880.yml

```
@@ -106,8 +117,9 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer: 
-  name: 
+  answer: The mistake was made in the networksubsystem of chromium. The original file changed for the fix  is ('net/http_utils..).
```

please make this a multi-line comment
```suggestion
  answer: |
    The mistake was made in the networksubsystem of chromium. The original file changed for the fix  is 
    ('net/http_utils..).
```